### PR TITLE
New Event & 3rd Party List Section

### PR DESCRIPTION
**Pull request by adamzero1**
- added `customer.login` event
- added a "3rd Party" section to the README to hopefully make it easier for people to share other common events, that may not belong in this repo.

### CODE REVIEW

**Review by gowrizrh — APPROVED** [review]

**Comment by gowrizrh** [comment]
LGTM :heavy_check_mark:

File: Observer/CustomerLoginObserver.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+class CustomerLoginObserver implements ObserverInterface
+{
+
+    public function __construct(
+        private readonly PublishingService $publishingService
+    ) {
+    }
+
+    /**
+     * @see @event customer_save_after
```
Comment (avstudnitz) [inline Observer/CustomerLoginObserver.php:20]: Please change the comment to `customer_login`.

**Review by avstudnitz — CHANGES_REQUESTED** [review]
Great addition, thanks! Just one change in the comment needed.
Also, the third party module list is great. Perhaps it would be even better to add a link to these modules on GitHub?

**Comment by avstudnitz** [comment]
Transferred the commit to #13

**Pull request closed without merging**